spec/connectors_sdk/sharepoint/extractor_spec.rb

```diff
@@ -293,42 +293,21 @@ def assert_drive_filtered(drive)
         expect_sites([:id => site_id])
         expect_groups([])
         expect_site_drives(site_id, drives)
+        allow(extractor).to receive(:retrieve_latest_cursors).and_return({ described_class::DRIVE_IDS_CURSOR_KEY => {} })
```

Comment:
is it worth adding any tests to make sure that `completed` is getting set in the right scenarios? The changes here look like you made existing tests pass, but not like you're testing your changes fixed anything.